### PR TITLE
Update nova files to be compliant with inspecs

### PR DESCRIPTION
**Pull request by edtubillara**

### CODE REVIEW

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by lihkin213** [comment]
retest

File: roles/nova-common/defaults/main.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ nova:
   # librbd1 version should match the ceph client version
   librbd1_version: "{{ ceph.client_version[ursula_os] }}"
   qemu_system_package: qemu-system-x86
-  glance_endpoint: http://{{ endpoints.main }}:9393
+  glance_endpoint: https://{{ endpoints.main }}:9393
```
Comment (nirajdp76) [inline roles/nova-common/defaults/main.yml]: we cannot change this to https. This is backend glance endpoint which is http. All our services backend ports are http.

Comment (lihkin213) [inline roles/nova-common/defaults/main.yml]: You may have to skip this check for this as well as other services

**Comment by edtubillara** [comment]
I'm going to close this PR since it will probably break things, I'll reopen it later.

**Pull request closed without merging**